_maps/map_files/Birdshot/birdshot.dmm

```diff
@@ -65949,6 +65949,11 @@
 /obj/effect/spawner/random/techstorage/engineering_all,
 /turf/open/floor/iron,
 /area/station/engineering/storage/tech)
+"wSP" = (
+/obj/effect/turf_decal/tile/neutral/fourcorners,
+/obj/structure/closet/secure_closet/blueshield,
+/turf/open/floor/iron/smooth,
+/area/station/command/bridge)
```

Comment:
I didn't look here.

Make an AM te4mplate this isn't how you do it.

Reply:
for single items, use the placer, not am